File: Services/Content/ContentAPI/Models/Dtos/ContentDto.cs
namespace ContentAPI.Models.Dtos;

public class ContentDto
{
    public string Id { get; set; }
    public string Title { get; set; }
    public string Text { get; set; }
    public List<Comment> Comments { get; set; }
    public List<Like> Likes { get; set; }
    public User User { get; set; }
    public string ImageUrl { get; set; }
    public string CategoryId { get; set; }
    
}

File: UI/MyBlogClient/src/app/modules/content/models/contentDto.ts
import { User } from "src/app/core/models/user"
import { CommentDto } from "./commentDto"
import { LikesDto } from "./likesDto"


export class ContentDto {
    id!: string
    title!: string
    text!: string
    comments:CommentDto[]=[]
    likes:LikesDto[]=[]
    user!:User
    imageUrl!:string
    categoryId!:string
}